src/DynamoCore/Graph/Workspaces/PackageDependencyInfo.cs

```diff
@@ -3,6 +3,18 @@
 
 namespace Dynamo.Graph.Workspaces
 {
+    /// <summary>
+    /// Enum containing the different types of package dependency states.
+    /// </summary>
+    internal enum PackageDependencyState
+    {
+        Loaded,
```

Comment:
@reddyashish - would mind adding ///summarys for each enum type to describe what it means a bit more specifically?

Reply:
Added.